app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -1259,6 +1259,19 @@ export default Controller.extend({
       if (opts.topicBody) {
         this.model.set("reply", opts.topicBody);
       }
+
+      let defaultComposerHeight;
+
+      if (this.model.action === "reply") {
+        defaultComposerHeight = "300px";
+      } else {
+        defaultComposerHeight = "400px";
+      }
```

Comment:
```suggestion
      const defaultComposerHeight = this.model.action === "reply" ? "300px" : "400px";
```

That should work too (it won't pass linting, though).